src/app/login/page.tsx

```diff
@@ -1,9 +1,11 @@
 'use client';
 
-import React, { useState } from 'react';
-import { signIn } from 'next-auth/react';
+import { useState } from 'react';
 import { useRouter } from 'next/navigation';
 
+import { authUser } from '../../client';
+import { useAuth } from '../../context/AuthContext';
+
 const LoginPage = () => {
   const [formData, setFormData] = useState({
```

Comment:
Let's make the next ticket about implementing the form without useState. It will be valuable lesson and is the more standard way I believe

Reply:
task created! :)